app/controllers/batch_connect/session_contexts_controller.rb

```diff
@@ -22,16 +15,12 @@ def new
         #{@app.validation_reason} Please contact support if you see this message
       EOT
     end
-  rescue BatchConnect::App::AppNotFound => e
```

Comment:
Since you removed this, you will need to add periods `.` to the end of all errors. So the error messages have proper-ish grammar.

Currently they will see something like this...

```
This app does not supply a form file under the directory '/path/to/app' Please contact support if you see this message
```

We should probably put a period between the error message and the "Please contact...".